features/mesh/dataplanes/Index.feature

```diff
@@ -39,16 +39,17 @@ Feature: mesh / dataplanes / index
 
     When I visit the "/mesh/default/data-planes" URL
 
-    Then the "$table-header" element exists 8 times
+    Then the "$table-header" element exists 9 times
     And the "$table-header" elements contain
-      | Value        |
-      | Name         |
-      | Service      |
-      | Protocol     |
-      | Zone         |
-      | Last Updated |
-      | Status       |
-      | Warnings     |
+      | Value              |
+      | Name               |
+      | Service            |
+      | Protocol           |
+      | Zone               |
+      | Last Updated       |
+      | Certificate Expiry |
```

Comment:
I'd call this `Certificate Info` as you are going to add more info here.

Reply:
Also when there's no cert maybe it's worth filling the column with `No certificate` or something like this.